src/media/class-wpml-page-builders-media-hooks.php

```diff
@@ -32,4 +33,25 @@ public function add_media_updater( $updaters ) {
 		}
 		return $updaters;
 	}
+
+	/**
+	 * @param string  $content
+	 * @param WP_Post $post
+	 *
+	 * @return string
+	 */
+	public function add_package_strings_content( $content, $post ) {
```

Comment:
@strategio I think it would be a cleaner design if this function was just a private function in `WPML_Media_Post_With_Media_Files`. I don't think there's any real need to call it via a filter.

Reply:
@brucepearson, intially I did this in WPML Media, but I figured out that this is only needed for metadata page builders and this is not needed for the ones using shortcodes.

Reply:
@strategio I thought we were handling shortcode strings in pretty much the same way as builders that have strings in metadata. If there is no simpler way or more unified way then I'm ok with your solution.

Reply:
@strategio On second thoughts, doesn't this code also run for page builders that use shortcodes?

Reply:
No @brucepearson, it runs only for metadata page builders.

For shortcodes page builders, the `<img>` tags are already inside `$post->post_content`.

In the next task, we will look in media modules (not in strings) to find other possible media usages, and this will apply to both metadata and shortcodes page builders.

Reply:
OK, thanks for the clarification.